src/app/shared/models/time.model.ts

```diff
@@ -1,6 +1,6 @@
 export interface TimeFrame {
-  startDateTime: number
```

Comment:
Shouldn't this be date? or a timestamp related type?

Reply:
Yes, I've fixed this.